<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>200</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>train.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QStackedWidget" name="pages">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>801</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(108, 202, 253);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page1">
     <widget class="QPushButton" name="admin">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>110</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 italic 18pt &quot;Ubuntu Mono&quot;;
background-color: rgba(197, 232, 126,1);
</string>
      </property>
      <property name="text">
       <string>ADMIN LOGIN</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="guest">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>180</y>
        <width>168</width>
        <height>36</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 italic 12pt &quot;Ubuntu Mono&quot;;
background-color: rgba(197, 232, 126,1);
</string>
      </property>
      <property name="text">
       <string>Sign in as Guest</string>
      </property>
     </widget>
     <widget class="QPushButton" name="quit">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>360</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 italic 20pt &quot;Ubuntu Mono&quot;;
background-color: rgba(197, 232, 126,1);
</string>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="login_page">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>49</y>
        <width>311</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Sign in </string>
      </property>
      <widget class="QLineEdit" name="username">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>151</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="password">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>100</y>
         <width>151</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_user">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_pass">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>58</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="submit_login">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>150</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 bold italic 14pt &quot;Ubuntu Mono&quot;;
background-color: rgba(197, 232, 126,1);
</string>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
      <widget class="QLabel" name="error_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>281</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 bold italic 14pt &quot;Ubuntu Mono&quot;;
</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ef2929;&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="back_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 bold italic 12pt &quot;Ubuntu Mono&quot;;
background-color: rgba(197, 232, 126,1);
</string>
      </property>
      <property name="text">
       <string>&lt;-- Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page2">
     <widget class="QCommandLinkButton" name="logout">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>40</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 italic 12pt &quot;Ubuntu Mono&quot;;
background-color: rgba(197, 232, 126,1);
</string>
      </property>
      <property name="text">
       <string>Log out</string>
      </property>
     </widget>
     <widget class="QLabel" name="welcome_message">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>211</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 bold italic 20pt &quot;Ubuntu Mono&quot;;
</string>
      </property>
      <property name="text">
       <string>Hello, Admin!</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="date_time">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>751</width>
        <height>91</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>10</number>
      </property>
     </widget>
     <widget class="QComboBox" name="train_ID">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>79</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>IR1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>IR2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>IR3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>IR4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>IR5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RE1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RE2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RE3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>180</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="delay">
      <property name="geometry">
       <rect>
        <x>155</x>
        <y>200</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 italic 12pt &quot;Ubuntu Mono&quot;;
background-color: rgba(197, 232, 126,1);
</string>
      </property>
      <property name="text">
       <string>0 minutes</string>
      </property>
     </widget>
     <widget class="QLabel" name="labe_ID">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>123</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 bold italic 12pt &quot;Ubuntu Mono&quot;;
</string>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_delay">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>180</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 bold italic 12pt &quot;Ubuntu Mono&quot;;

</string>
      </property>
      <property name="text">
       <string>Delay:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="submit_delay">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>250</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 bold italic 14pt &quot;Ubuntu Mono&quot;;
background-color: rgba(197, 232, 126,1);
</string>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page3">
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>761</width>
        <height>211</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>759</width>
         <height>209</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="info_label">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QCommandLinkButton" name="departures">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Check the next hour departures</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="arrivals">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Check the next hour arrivals</string>
      </property>
     </widget>
     <widget class="QLabel" name="schedule_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 bold italic 20pt &quot;Ubuntu Mono&quot;;

</string>
      </property>
      <property name="text">
       <string>Today's Schedule:</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="bactToPage1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>380</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 bold italic 12pt &quot;Ubuntu Mono&quot;;
background-color: rgba(197, 232, 126,1);
</string>
      </property>
      <property name="text">
       <string>&lt;-- Back</string>
      </property>
     </widget>
     <zorder>departures</zorder>
     <zorder>arrivals</zorder>
     <zorder>schedule_label</zorder>
     <zorder>bactToPage1</zorder>
     <zorder>scrollArea</zorder>
    </widget>
    <widget class="QWidget" name="page4">
     <widget class="QPushButton" name="backToPage3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>380</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 bold italic 12pt &quot;Ubuntu Mono&quot;;
background-color: rgba(197, 232, 126,1);
</string>
      </property>
      <property name="text">
       <string>&lt;-- Back</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>771</width>
        <height>271</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>769</width>
         <height>269</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="next_hour_label">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777179</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font:75 italic 12pt &quot;Ubuntu Mono&amp;quot&quot;;


</string>
            </property>
            <property name="text">
             <string>Next Hour Trains Info Here</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <zorder>pages</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
